server/services/search.js

```diff
@@ -15,7 +15,7 @@ const createSearchService = ({
   }
 
   function typeAhead({ query, limit, establishmentId }) {
-    const prison = getEstablishmentFormattedName(establishmentId);
+    const prison = getEstablishmentSearchName(establishmentId);
```

Comment:
Same query as above...
Do we set `prison` first to highlight errors? Otherwise, it should probably go below the `query === ''` check following the whole return early ideology.

Reply:
same as above.. totally agree but think we can leave for now?